rust/ql/lib/codeql/rust/frameworks/stdlib/lang-core.model.yml

```diff
@@ -11,6 +11,8 @@ extensions:
       - ["<_ as core::convert::Into>::into", "Argument[self].Reference.Element", "ReturnValue.Element", "taint", "manual"]
       - ["<alloc::string::String as core::convert::Into>::into", "Argument[self].Element", "ReturnValue.Element", "taint", "manual"]
       - ["<alloc::string::String as core::convert::Into>::into", "Argument[self].Reference.Element", "ReturnValue.Element", "taint", "manual"]
+      # From
+      - ["<_ as core::convert::From>::from", "Argument[0]", "ReturnValue", "value", "manual"]
```

Comment:
I would have expected that this should be a `taint` model and not a `value` model?